### DISCUSSION TITLE
arduino micro

### DISCUSSION THREAD

**Discussion by djsedaw** [BIPES/BIPES#125] [Q&A]
Would BIPES work with an arduino micro?

**Comment by rafaelaroca** [comment]
Hello, yes. It is based on the ATmega32u4, which is pretty similar to the AtMega328 used in the Uno and Nano. Micro has a built-in USB port, so, maybe our automatic upload tool might not work:

https://bipes.net.br/snek-web-uploader/

However, if you flash SNEK on the Micro, it will work!
Here it is the support for it: https://keithp.com/pipermail/snek/2019-May/000154.html

regards
Rafael

**Reply by djsedaw** [reply]
Thanks, I will give it a go!
